3rdparty/voice_text/README.md

```diff
@@ -0,0 +1,55 @@
+voice_text
+==========
+
+ROS Interface for HOYA VoiceText Speech Synthesis Engine
+
+## Installation
+
+1. Install Library
+
+```bash
```

Comment:
These instructions are related with undisclosed information, e.g., closed voicetext manuals.
In addition, using `tar` seems not to be official instruction of voicetext installation.
I think these information can be removed in this open repository.